src/OpenTK.GraphicsLibraryFramework/Enums/ClientApi.cs

```diff
@@ -0,0 +1,24 @@
+namespace OpenToolkit.GraphicsLibraryFramework
+{
+    /// <summary>
+    /// The context client apis.
```

Comment:
"apis" should have the first three letters capitalized

Reply:
Same for something like "ContextClientAPIAttribute"?
or do you mean just for the documentation text?

Reply:
I just mean for the doc text, the actual enum names are fine.